src/test/java/com/wikia/webdriver/testcases/mercurytests/curatedcontenttests/ImageUploadTests.java

```diff
@@ -26,34 +26,31 @@
   @BeforeMethod(alwaysRun = true)
   public void beforeMethod() {
     wikiURL = urlBuilder.getUrlForWiki(MercuryWikis.MERCURY_EMPTY_CC_EDITOR);
-
     new CuratedContent().clear();
-
-    // This login is temporary solution, use @Execute after QAART-669 is done
-    new LoginPageObject(driver).get().logUserIn(Configuration.getCredentials().userNameStaff2,
-            Configuration.getCredentials().passwordStaff2);
   }
 
   @Test(groups = "MercuryImageUploadTest_001")
   @Execute(onWikia = MercuryWikis.MERCURY_EMPTY_CC_EDITOR,
-          allowedDriver = Driver.ChromeMobileMercury)
+      allowedDriver = Driver.ChromeMobileMercury,
+      asUser = User.STAFF)
   public void MercuryImageUploadTest_001_saveNewPhoto() {
-      CuratedMainPagePageObject curatedMainPagePageObject = new CuratedMainPagePageObject(driver);
-      EditorHomePageObject editorHomePageObject = new EditorHomePageObject(driver);
-
-      curatedMainPagePageObject.isCuratedContentVisible();
-      curatedMainPagePageObject.navigateToUrlWithPath(wikiURL, MercuryPaths.ROOT_MAIN_EDIT);
-
-      ImageGenerator generator = new ImageGenerator();
-      generator.generateImageWithRandomText();
-      String imagePath = generator.getImageAbsolutePath();
-
-      ItemFormPageObject item = editorHomePageObject.clickAddFeaturedContent();
-      UploadImageModalComponentObject upload = item.clickOnImage();
-      CroppingToolPageObject
-              crop =
-              upload.uploadImage(
-                      imagePath);
-      crop.waitForCropperToBeLoaded();
+    CuratedMainPagePageObject curatedMainPagePageObject = new CuratedMainPagePageObject(driver);
+    EditorHomePageObject editorHomePageObject = new EditorHomePageObject(driver);
+    ;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)